starlette/testclient.py

```diff
@@ -68,6 +68,18 @@ def __init__(self, session: "WebSocketTestSession") -> None:
         self.session = session
 
 
+class DenialResponse(Exception):
```

Comment:
nit: maybe make it clear it is websocket-only exception?

```suggestion
class WebsocketDenialResponse(Exception):
```